src/components/editor/EditorAppController.jsx

```diff
@@ -120,7 +120,7 @@ export default class EditorAppController extends BaseComponent {
   }
 
   render() {
-    const navItems = ['Articles', 'Authors', 'Issues', 'Images'];
+
```

Comment:
And you're so close! I'm happy with everything now, but you just have a linting error here that seems to be because you started this block with a new line :)